powa_collector/__init__.py

```diff
@@ -42,11 +42,14 @@
 
 
 def getVersion():
+    """Return powa_collector's version as a string"""
     return __VERSION__
 
 
 class PowaCollector():
+    """Main powa collector's class. This manages all collection tasks"""
     def __init__(self):
+        """Instance creator. Sets logging, signal handlers, and basic structure"""
```

Comment:
Should we mention that *all* attributes should be explicitly declared here, and that we won't (don't want to) have dynamic attributes?